app/components/MissionSection.tsx

```diff
@@ -31,10 +30,7 @@ export default function MissionSection() {
 					>
 						Local Tech Innovation
 					</Typography>
-					<Typography
-						variant="lead"
-						className="text-[#0C1F14] lg:w-[69%] text-[22px]"
-					>
+					<Typography className="text-[#0C1F14] lg:w-[69%] text-[22px]">
```

Comment:
The color is already defined in the theme, so you should be able to use it with `text-secondary-10`

Reply:
This content is not relevant for this PR. Resolving as this will be fixed in an upcoming PR